plugins/pack-deploy.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: pack-deploy
+spec:
+  version: "v0.1.0"
+  platforms:
+  - uri: https://github.com/dougsland/kubectl-pack-deploy/archive/kubectl-pack-deploy-0.1.0.tar.gz
+    sha256: 403b04620f47c8d67f09c0aa405723fc9acdf3cc98face60f4adb9ff8e358d7e
+    bin: kubectl-pack_deploy
+    files:
+    - from: "./*/kubectl-pack_deploy"
+      to: .
+    - from: "./*/LICENSE"
+      to: .
+    selector:
+      matchExpressions:
+      -   {key: os, operator: In, values: [darwin, linux]}
+  caveats: |
+    This plugin needs the following programs:
+    * kubectl neat plugin
```

Comment:
@itaysk may find this interesting :)

Reply:
Oh yeah, really liked kubectl neat. kudos to @itaysk

Reply:
awesome! 😃